CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+## [1.3.0] - 2018-06-13
+
+* Updated color names in InVision DSM
```

Comment:
Let's mention it fixes #16 and it is a breaking change so we'll need to bump up to `2.0.0`.